TI_Practice4/Pactice4.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace TI_Practice4
+{
+    class Practice4
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Please write something.");
+            String input = Console.ReadLine();
+
+            Palindrome palindrome1 = new Palindrome();
+            Palindrome palindrome2 = new Palindrome();
+
+            if (input.Length > 1)
+            {
+                if (palindrome1.ToAlphaNumeric(input).Equals(palindrome2.GetInvertedString(input)))
```

Comment:
it is a creative solution, thanks 🎉 

but just so you know it is also possible to do it using a single method.
you can try to implement such a solution as well if you want